src/js/engagement_view/src/components/engagementView/sidebar/tables/toggleLensTable.tsx

```diff
@@ -48,29 +49,27 @@ export function ToggleLensTable({ setLens }: ToggleLensTableProps) {
         setPageState(0);
     };
 
+    async function updateTableStateWithLenses() {
```

Comment:
This function is exactly the same as the anonymous setInterval function it replaces (but also ported to await())